test/container/dummy/Dockerfile

```diff
@@ -5,47 +5,50 @@ ENV DEBIAN_FRONTEND="noninteractive"
 
 ARG PYTHON=python3
 ARG PIP=pip3
-ARG PYTHON_VERSION=3.6.8
+ARG PYTHON_VERSION=3.7.10
 
 RUN apt-get update \
- && apt-get install -y --no-install-recommends \
-    build-essential \
-    ca-certificates \
-    wget \
-    zlib1g-dev \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/*
+   && apt-get install -y --no-install-recommends \
+   build-essential \
+   ca-certificates \
+   wget \
+   zlib1g-dev \
+   && apt-get clean \
+   && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update \
+   && apt-get install -y --no-install-recommends \
+   libbz2-dev \
+   libc6-dev \
+   libffi-dev \
+   libgdbm-dev \
+   liblzma-dev \
+   libncursesw5-dev \
+   libreadline-gplv2-dev \
+   libsqlite3-dev \
+   libssl-dev \
+   tk-dev \
+   ffmpeg \
+   libsm6 \
+   libxext6 \
+   && rm -rf /var/lib/apt/lists/* \
+   && apt-get clean
 
 RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz \
- && tar -xvf Python-$PYTHON_VERSION.tgz \
- && cd Python-$PYTHON_VERSION \
- && ./configure \
- && make \
- && make install \
- && apt-get update \
- && apt-get install -y --no-install-recommends \
-    libreadline-gplv2-dev \
-    libncursesw5-dev \
-    libssl-dev \
-    libsqlite3-dev \
-    libgdbm-dev \
-    libc6-dev \
-    libbz2-dev \
-    tk-dev \
- && make \
- && make install \
- && rm -rf ../Python-$PYTHON_VERSION* \
- && ln -s /usr/local/bin/pip3 /usr/bin/pip \
- && ln -s $(which ${PYTHON}) /usr/local/bin/python \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/*
+   && tar -xvf Python-$PYTHON_VERSION.tgz \
+   && cd Python-$PYTHON_VERSION \
+   && ./configure && make && make install \
+   && make && make install && rm -rf ../Python-$PYTHON_VERSION*
```

Comment:
Duplicate "make" and "make install" commands here, so it's building python twice.